frontend/portals/GlobalCssRules/index.jsx

```diff
@@ -5,4 +5,8 @@ css.global('#bolt-tracking-frame', {
   display: 'none',
 });
 
+css.global('#bolt-modal-background', {
+  top: 'env(safe-area-inset-top)',
```

Comment:
You have to do it this way:
```suggestion
  top: 'var(--safe-area-inset-top)',
```
because of this https://github.com/shopgate/pwa/blob/f7292116f62fcb83765875b86e206ac0929bdd34/themes/theme-ios11/components/Viewport/style.js#L54-L68

Reply:
done